src/models-spec.js

```diff
@@ -103,15 +114,23 @@ describe('models', () => {
               if (!result.pass) {
                 result.message = `${typeRef} is not a valid schema.org reference`;
               }
+            } else if (typeId.match(/^http:\/\/purl\.org\/goodrelations\/v1/)) {
+              result.pass = goodRelationsDataModel.includes(typeId);
+              if (!result.pass) {
+                result.message = `${typeRef} is not a valid goodrelations reference`;
+              }
+            } else if (typeId.match(/^http:\/\/www\.w3\.org\/2004\/02\/skos\/core#/)) {
+              result.pass = skosDataModel.includes(typeId);
+              if (!result.pass) {
+                result.message = `${typeRef} is not a valid goodrelations reference`;
+              }
             } else if (typeId.match(/^https:\/\/openactive.io/)) {
               const typeName = typeId.replace(/^https:\/\/openactive.io\//, '');
               const enums = getEnums(version);
               result.pass = modelExists(typeName) || Object.prototype.hasOwnProperty.call(enums, typeName);
               if (!result.pass) {
-                result.message = `${typeRef} is not a valid Open Active reference`;
```

Comment:
Note OpenActive never has a space between “Open” and “Active”

Reply:
Bad Henry. Thanks, @nickevansuk. Fixing this in later PR.

Reply:
No worries